src/registers/xcontrol.rs

```diff
@@ -0,0 +1,95 @@
+//! Access to various extended system registers
+use bitflags::bitflags;
+
+/// Extended feature enable mask register
+#[derive(Debug)]
+pub struct XCr0;
+
+bitflags! {
+    /// Configuration flags of the XCr0 register.
+    pub struct XCr0Flags: u64 {
+        /// Enables x87 FPU
+        const X87 = 1;
+        /// Enables 128-bit (legacy) SSE
+        /// Must be set to enable AVX and YMM
+        const SSE = 1<<1;
+        /// Enables 256-bit SSE
+        /// Must be set to enable AVX
+        const YMM = 1<<2;
+        /// When set, PKRU state management is supported by
+        /// ZSAVE/XRSTOR
+        const MPK = 1<<9;
```

Comment:
[Wikipedia](https://en.wikipedia.org/wiki/Control_register#XCR0_and_XSS) names this flag `PKRU`. I'm not sure what `MPK` stands for?

Reply:
![1617301804_grim](https://user-images.githubusercontent.com/22085373/113338272-2e9e4a80-9329-11eb-98a9-109329cd7fed.png)

I just named it like in the AMD Documentation

Reply:
Ah, makes sense, thanks!